infra/gcp/ensure-main-project.sh

```diff
@@ -79,6 +84,40 @@ ensure_private_gcs_bucket \
     "${PROJECT}" \
     "gs://${CLUSTER_TERRAFORM_BUCKET}"
 
+
+color 6 "Ensuring all the prow buckets exist"
```

Comment:
/hold
The rest of this I would like to wait to merge until I land a refactor of this file (https://github.com/kubernetes/k8s.io/pull/1974)

I am happy to take up redoing this part to fit in with the refactor as followup, or leaving it to you to ensure someone other than me understands the structure I tried to put in place

Reply:
@spiffxp made a refactor now #1974 is merged.